index.php

```diff
@@ -1,15 +1,9 @@
 <?php
-
-define('ROOT_DIR', realpath(dirname(__FILE__)) . '/');
-define('LIB_DIR', ROOT_DIR . 'lib/');
-define('VENDOR_DIR', ROOT_DIR . 'vendor/');
-define('PLUGINS_DIR', ROOT_DIR . 'plugins/');
-define('THEMES_DIR', ROOT_DIR . 'themes/');
-define('CONFIG_DIR', ROOT_DIR . 'config/');
-define('CACHE_DIR', LIB_DIR . 'cache/');
-
-define('CONTENT_EXT', '.md');
-
-require_once(VENDOR_DIR . 'autoload.php');
-require_once(LIB_DIR . 'pico.php');
-$pico = new Pico();
+require_once(__DIR__ . '/vendor/autoload.php');
+$pico = new Pico(
+    __DIR__,
+    'config/',
```

Comment:
Providing an easy way to integrate Pico in other applications was the intention of removing the constants, so this definitely is a completely fair use case. It is not necessary to move the whole config handling to `index.php`, it's entirely sufficient to provide an API to do such things - so I've added a `Pico::setConfig()` method. See method docs for details.